src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -37,10 +37,11 @@ public function __construct(LoggerInterface $log)
 	/**
 	 * Report or log an exception.
 	 *
+	 * @param  \Illuminate\Http\Request  $request
 	 * @param  \Exception  $e
 	 * @return void
 	 */
-	public function report(Exception $e)
+	public function report($request, Exception $e)
```

Comment:
Yeah, but it could give a false positive output if you try to log the URL for none HTTP related exception. As a default dummy request object is always available on every console execution. 

https://github.com/laravel/framework/blob/master/src/Illuminate/Foundation/Bootstrap/SetRequestForConsole.php